src/interfaces/Relationship.ts

```diff
@@ -0,0 +1,7 @@
+export interface Relationship {
```

Comment:
Not sure if we need to define this. Orbit already has TypeScript typings for schema.